client/my-sites/email/email-providers-comparison/in-depth/select-button.tsx

```diff
@@ -29,7 +29,11 @@ const usePlanAvailable = (
 		return true;
 	}
 
-	if ( ! canPurchaseGSuite || ! hasGSuiteSupportedDomain( [ domain ] ) ) {
+	if ( ! canPurchaseGSuite ) {
+		return false;
+	}
+
+	if ( ! domain || ! hasGSuiteSupportedDomain( [ domain ] ) ) {
```

Comment:
This fixes an error triggered when loading the `In-Depth Comparison` page (when the list of domains has not been retrieved yet):

<img width="542" alt="ERROR" src="https://user-images.githubusercontent.com/594356/158850112-3d7694f5-e05e-4d66-9566-5ce56e9f25bd.png">